### PR TITLE
Re-pin lock files so they have the new services element

### PR DESCRIPTION
**Pull request by cheister**
Regenerated lock files after https://github.com/bazelbuild/rules_jvm_external/pull/1063 was merged

### CODE REVIEW

File: tests/custom_maven_install/regression_testing_install.json [hunk 1]
```diff
@@ -5816,6 +5816,209 @@
...
+        "org.apache.flink.shaded.jackson2.com.fasterxml.jackson.dataformat.csv.CsvFactory",
+        "org.apache.flink.shaded.jackson2.com.fasterxml.jackson.dataformat.yaml.YAMLFactory"
+      ],
+      "org.apache.flink.shaded.jackson2.com.fasterxml.jackson.core.ObjectCodec": [
+        "org.apache.flink.shaded.jackson2.com.fasterxml.jackson.databind.ObjectMapper"
+      ]
+    },
+    "org.apache.kafka:kafka-clients": {
+      "org.apache.kafka.common.config.provider.ConfigProvider": [
+        " #",
```
Comment (cheister) [inline tests/custom_maven_install/regression_testing_install.json:6000]: @illicitonion Is this comment with the license expected output?

Comment (illicitonion) [reply tests/custom_maven_install/regression_testing_install.json:6000]: Ooh, no it's not, will send a PR in a sec!

Comment (illicitonion) [reply tests/custom_maven_install/regression_testing_install.json:6000]: https://github.com/bazelbuild/rules_jvm_external/pull/1101 - thanks for the catch!

**Review by shs96c — APPROVED** [review]
LGTM! Ship it!

**Pull request merged**